.github/lf-scale-config.yml

```diff
@@ -343,3 +343,27 @@ runner_types:
     os: linux
     variants:
       ephemeral: {}
+  lf.linux.8xlarge.amd:
+    disk_size: 200
+    instance_type: m7a.8xlarge
+    is_ephemeral: true
+    os: linux
+    variants:
+      ephemeral:
+        is_ephemeral: true
+  lf.linux.12xlarge.amd:
+    disk_size: 200
+    instance_type: m6a.12xlarge
+    is_ephemeral: true
+    os: linux
+    variants:
+      ephemeral:
+        is_ephemeral: true
+  lf.linux.24xl.amd:
+    disk_size: 200
+    instance_type: m7a.24xlarge
+    is_ephemeral: true
+    os: linux
+    variants:
+      ephemeral:
+        is_ephemeral: true
```

Comment:
@jeanschmidt , @ZainRizvi : do those needs to be always in sync? I.e. if someone adds runners to so far create test PR, shoudl they be added only to scale-config.yml and not to LF one?

Reply:
From the CI Sync meeting just now. Yes the 4 files do need to be in sync in order to work. It is also possible to exclude the jobs from running in LF runners by runner determinator configuration in the job config. Its a feature we just added on Friday.

Here's an example of inductor jobs opting out of LF runners.

https://github.com/pytorch/pytorch/pull/153151/files